### PR TITLE
fix: user detail broken in demo (#59)

### PR DESCRIPTION
**Pull request by Almas-Ali**
As the issue #59 mentioned about the broken user detail page. I fixed the page and fixed the url from `/more/{id}` to `/table/users/{id}`. Now, It looks more relevant than `/more` route.

Hope it make sense.

### CODE REVIEW

File: python/demo/tables.py [hunk 1]
```diff
@@ -135,3 +135,22 @@ def tabs() -> list[AnyComponent]:
             class_name='+ mb-4',
         ),
     ]
+
+
+@router.get('/users/{id}/', response_model=FastUI, response_model_exclude_none=True)
+def user_profile(id: int) -> list[AnyComponent]:
+    users = [
```
[the remark this answers is not in the document]
Comment (Almas-Ali) [reply python/demo/tables.py]: I have resolved this and changed the class name `MyTableRow` to `User` for better understanding. Also added `fields` in city details.

[the remark this answers is not in the document]
Comment (Almas-Ali) [reply python/demo/tables.py]: `users: list[User]` is now a global variable to access from everywhere.

File: python/demo/tables.py [hunk 2]
```diff
@@ -135,3 +135,22 @@ def tabs() -> list[AnyComponent]:
...
+        MyTableRow(id=3, name='Jack', dob=date(1992, 1, 1)),
+    ]
+    try:
+        user = next(user for user in users if user.id == id)
+    except StopIteration:
+        raise HTTPException(status_code=404, detail='User not found')
+    return demo_page(
+        *tabs(),
+        c.Link(components=[c.Text(text='Back')], on_click=BackEvent()),
+        c.Details(data=user),
```
[the remark this answers is not in the document]
Comment (Almas-Ali) [reply python/demo/tables.py]: Done adding `fields` here and also added in city details for better explicit controls.

File: python/demo/tables.py [hunk 3]
```diff
@@ -135,3 +135,22 @@ def tabs() -> list[AnyComponent]:
             class_name='+ mb-4',
         ),
     ]
+
+
+@router.get('/users/{id}/', response_model=FastUI, response_model_exclude_none=True)
+def user_profile(id: int) -> list[AnyComponent]:
```
[the remark this answers is not in the document]
Comment (Almas-Ali) [reply python/demo/tables.py:159]: `list[...]` is mentioned everywhere. It won't effect on just a single line of change.

[the remark this answers is not in the document]
Comment (samuelcolvin) [reply python/demo/tables.py]: `list[]` is fine here since the demo runs on 3.11 only.

**Comment by samuelcolvin** [comment]
this is great, thanks so much.

**Pull request merged**